src/jit/compiler.h

```diff
@@ -1172,8 +1215,10 @@ class  fgArgInfo
     void            RecordStkLevel     (unsigned        stkLvl);
     unsigned        RetrieveStkLevel   ();
 
-    unsigned            ArgCount ()  { return argCount; }
-    fgArgTabEntryPtr *  ArgTable ()  { return argTable; }
+    unsigned            ArgCount ()      { return argCount; }
+    fgArgTabEntryPtr *  ArgTable ()      { return argTable; }
```

Comment:
Jit CC: T* instead of T *

e.g. fgArgTabEntryPtr*

(Similar correction is needed on line 1131 - i will leave it up to you whether you want to correct that occurrence as well)